server/src/document/document.controller.ts

```diff
@@ -68,4 +71,12 @@ export class DocumentController {
   async destroy(@Query('serverRelativeUrl') serverRelativeUrl) {
     return this.crmService.deleteSharepointFile(serverRelativeUrl);
   }
+
```

Comment:
Would prefer some comments here documenting the `path` parameter, with an example path! 
Otherwise, looks good